sensor_msgs/JointState        joint_states
geometry_msgs/PointStamped[]  rgbd_observations
geometry_msgs/PointStamped[]  world_observations
DepthCameraInfo               rgbd_info
sensor_msgs/PointCloud2       cloud
